build.gradle

```diff
@@ -275,6 +275,7 @@ ext.schedulingFunctionalTestConfiguration = {
 ext.schedulingPerformanceTestConfiguration = {
 
     include 'performancetests/recovery/**'
+    include 'performancetests/metrics/**'
```

Comment:
maybe you can reuse the same logic I used for functional tests, i.e. create a test suite class like PerformanceTestSuite, and use it to group the tests together